cbindings/logic/acp.go

```diff
@@ -0,0 +1,209 @@
+	targetActorArg string,
+	TxnID uint64,
+) GoCResult {
+	ctx := context.Background()
+
+	ctx, err := contextWithIdentity(ctx, identityPrivateKey)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	ctx, err = contextWithTransaction(n, ctx, TxnID)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	result, err := GetNode(n).DB.AddDACActorRelationship(ctx, collectionArg, docIDArg, relationArg, targetActorArg)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	return marshalJSONToGoCResult(result)
+}
+
+func ACPDeleteDACActorRelationship(
+	n int,
+	identityPrivateKey string,
+	collectionArg string,
+	docIDArg string,
+	relationArg string,
+	targetActorArg string,
+	TxnID uint64,
+) GoCResult {
+	ctx := context.Background()
+
+	ctx, err := contextWithIdentity(ctx, identityPrivateKey)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	ctx, err = contextWithTransaction(n, ctx, TxnID)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	result, err := GetNode(n).DB.DeleteDACActorRelationship(ctx, collectionArg, docIDArg, relationArg, targetActorArg)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	return marshalJSONToGoCResult(result)
+}
+
+func ACPNodeDisable(n int, identityPrivateKey string, TxnID uint64) GoCResult {
+	ctx := context.Background()
+
+	ctx, err := contextWithIdentity(ctx, identityPrivateKey)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	ctx, err = contextWithTransaction(n, ctx, TxnID)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	if err := GetNode(n).DB.DisableNAC(ctx); err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	return marshalJSONToGoCResult(client.SuccessResponse{Success: true})
+}
+
+func ACPNodeReEnable(n int, identityPrivateKey string, TxnID uint64) GoCResult {
+	ctx := context.Background()
+
+	ctx, err := contextWithIdentity(ctx, identityPrivateKey)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	ctx, err = contextWithTransaction(n, ctx, TxnID)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	if err := GetNode(n).DB.ReEnableNAC(ctx); err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	return marshalJSONToGoCResult(client.SuccessResponse{Success: true})
+}
+
+func ACPNodeRelationshipAdd(
+	n int,
+	identityPrivateKey string,
+	relationArg string,
+	targetActorArg string,
+	TxnID uint64,
+) GoCResult {
+	ctx := context.Background()
+
+	ctx, err := contextWithIdentity(ctx, identityPrivateKey)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	ctx, err = contextWithTransaction(n, ctx, TxnID)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	addNACActorRelationshipResult, err := GetNode(n).DB.AddNACActorRelationship(ctx, relationArg, targetActorArg)
+	if err != nil {
+		return returnGoC(1, err.Error(), "")
+	}
+
+	return marshalJSONToGoCResult(addNACActorRelationshipResult)
+}
+
+func ACPNodeRelationshipDelete(
```

Comment:
Suggestion: now that you have fixed the name of `ACPDeleteDACActorRelationship` this should match similar style i.e. `ACPDeleteNACActorRelationship`